cardano-db/src/Cardano/Db/Query.hs

```diff
@@ -1233,34 +1242,36 @@ queryAdaPots blkId = do
 -- | Get the UTxO set after the specified 'BlockId' has been applied to the chain.
 -- Not exported because 'BlockId' to 'BlockHash' relationship may not be the same
 -- across machines.
-queryUtxoAtBlockId :: MonadIO m => BlockId -> ReaderT SqlBackend m [(TxOut, ByteString)]
+queryUtxoAtBlockId :: MonadIO m => BlockId -> ReaderT SqlBackend m [(TxOut, Text, ByteString)]
```

Comment:
I'll need two versions of these queries, 
1: for use with`AdressDetail` and when config flag is true
2: normal use with no flag or use of `AddressDetail` table